vue/components/organisms/pickers/pickers.vue

```diff
@@ -496,7 +496,7 @@ export const Pickers = {
             this.multipleMaterials = false;
         },
         activePart(part, oldPart) {
-            if (!part) {
+            if (!part || !this.materialOptions) {

```

Comment:
Because the SDK's `config()` is not `await`ing for `setChoices`, therefore you might receive a `post_config` event before you receive the `choices` event for the same `config()` call. The result is that you can't access `mateiralOptions`, which is a by-product of `filteredOptions`, because the `choices` are empty.

In more abstract terms: if you don't have a viable UI option to show for a given material, then there's nothing more to do in this function.

In any case, I've changed the approach to this problem, it probably looks better now.